$NetBSD: patch-av,v 1.1.1.1 2008/04/24 01:50:58 tnn Exp $

Add a knob (enviroment variable) for disabling installation of egg metadata
in extensions until we have infrastructure in place for dealing w/ it.

--- Lib/distutils/command/install.py.orig	2006-03-27 23:55:21.000000000 +0200
+++ Lib/distutils/command/install.py
@@ -601,7 +601,8 @@ class install (Command):
                     ('install_headers', has_headers),
                     ('install_scripts', has_scripts),
                     ('install_data',    has_data),
-                    ('install_egg_info', lambda self:True),
                    ]
+    if not os.environ.has_key('PKGSRC_PYTHON_NO_EGG'):
+        sub_commands += [('install_egg_info', lambda self:True),]
 
 # class install
